app/assets/stylesheets/admin.scss

```diff
@@ -1 +1,9 @@
-@import "admin/*";
\ No newline at end of file
+@import "admin/bootstrap.css";
+@import "admin/bootstrap-multiselect.css";
+@import "admin/bootstrap-tagsinput.css";
+@import "admin/datapicker3.css";
+@import "admin/theme.css";
+@import "admin/default.css";
+@import "admin/fontawesome.scss.erb";
+@import "admin/jquery-ui-1.10.4.custom.css";
+@import "admin/magnific-popup.css";
```

Comment:
you did this because the order of import matter?

Reply:
because the order default must be after theme